ce/cli/commands/activate.ts

```diff
@@ -39,6 +41,11 @@ export class ActivateCommand extends Command {
     debug(i`Deactivating project ${projectFile(projectManifest.metadata.context.file)}`);
     await session.deactivate();
 
-    return await activateProject(projectManifest, this.commandLine);
+    return await activateProject(projectManifest, {
```

Comment:
I know this is wrong because it must be somehow stripping off which identifiers are to become "true" in activation but that doesn't appear to work before my changes either...